crates/ruff/src/printer.rs

```diff
@@ -27,11 +27,11 @@ bitflags! {
     #[derive(Default, Debug, Copy, Clone)]
     pub(crate) struct Flags: u8 {
         /// Whether to show violations when emitting diagnostics.
-        const SHOW_VIOLATIONS = 0b0000_0001;
+        const SHOW_VIOLATIONS = 1 << 0;
         /// Whether to show a summary of the fixed violations when emitting diagnostics.
-        const SHOW_FIX_SUMMARY = 0b0000_0100;
+        const SHOW_FIX_SUMMARY = 1 << 2;
```

Comment:
Is there any particular reason to not change this to `1 << 1`?

Reply:
It should be fine. I suspect that we used to have more flags and didn't update the values when we removed one.